packages/forklift-console-plugin/src/mock-console-extension/setupBrowserWorker.tsx

```diff
@@ -0,0 +1,46 @@
+import { type SetupWorker, rest, setupWorker } from 'msw';
+
+import { allHandlers } from '@kubev2v/mocks/console-handlers';
+
+export { type SetupWorker } from 'msw';
+
+const fancyLog = (url, ...rest) => [
+  '%cmock-plugin passthrough%c \u{1fa83} %s',
+  'font-weight: bold',
+  'font-weight: normal',
+  url,
+  ...rest,
+];
+
+/**
+ * Setup the MockServiceWorker with all of the mock handlers available handling request
+ * for UI url paths.  The worker will be added to `window.msw` for debugging convenience.
+ *
+ * @param staticFilePath URL prefix where `mockServiceWorker.js` will be available
+ * @returns A setup and started MockServiceWorker
+ */
+export function setupBrowserWorker(staticFilePath: string): SetupWorker {
+  const worker = setupWorker(
+    ...allHandlers,
```

Comment:
we should also log calls that maths mocked data,

IMHO we should see here all the paths we handle, e.g. if we handle `providers` we should see here the specific handler, the mock package should not care about what paths we support, only about getting the mock data for a specific path

Reply:
ok, the allHandlers now return empty array, we can refine this in some followup